README.md

```diff
@@ -1070,6 +1070,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Tawfik Yasser](https://github.com/TawfikYasser)
   - [Taye-Vaughn Jones](https://github.com/tvjones)
   - [Tejas Shah](https://github.com/Hero4440)
+  - [Teja Sai](https://github.com/tech-w-tej)
```

Comment:
Please add your name in line 1072, after Taye-Vaughn Jones.
Let us know if you need any help doing this! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient